openproblems/data/scicar_chrom/base.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import pandas as pd
+import scanpy as sc
+import anndata
+import scprep
+import tempfile
+import os
+
+
+def load_scicar(
```

Comment:
Isn't this just a duplicate of `data/multimodal/scicar`? Why can't you just use that?

Reply:
No, it is extended to fill the .var from peaks to contain the peak coordinates.